File: Examples/test-suite/perl5/memberin_extend_c_runme.pl
use strict;
use warnings;
use Test::More tests => 3;
BEGIN { use_ok('memberin_extend_c') }
require_ok('memberin_extend_c');

my $t = memberin_extend_c::Person->new();
$t->{name} = 'Fred Bloggs';
is($t->{name}, 'FRED BLOGGS');

File: Examples/test-suite/go/memberin_extend_c_runme.go
package main

import "swigtests/memberin_extend_c"

func main() {
	t := memberin_extend_c.NewPerson()
	t.SetName("Fred Bloggs")
	if t.GetName() != "FRED BLOGGS" {
		panic("name wrong")
	}
}
